app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -577,28 +579,46 @@ class ChatController(args: Bundle) : BaseController(args), MessagesListAdapter
         }
     }
 
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+    override fun onActivityResult(requestCode: Int, resultCode: Int, intent: Intent?) {
         if (requestCode == REQUEST_CODE_CHOOSE_FILE) {
             if (resultCode == RESULT_OK) {
-                uploadFile(data)
+                checkNotNull(intent)
```

Comment:
Is the resulting IllegalStateException somewhere handled?

Reply:
done